src/pages/index.js

```diff
@@ -0,0 +1,327 @@
+import { settings, enableValidation, resetValidation, disableButton } from "../scripts/validation.js";
+import "./index.css";
+import logo from '../images/logo.svg';
+import avatar from '../images/avatar.jpg';
+import pencilIcon from '../images/pencilicon.svg';
+import lightPencilIcon from '../images/whitepencil.svg';
+import plusIcon from '../images/button.svg';
+import Api from '../utils/Api.js';
+
+document.getElementById('header-logo').src = logo;
+document.getElementById('profile-avatar').src = avatar;
```

Comment:
`document.getElementById('profile-avatar')`

You've already found the avatar in line 57 as `profileAvatar`. Please, define it in line 9 to reuse it